backend/models/cans.py

```diff
@@ -34,11 +34,24 @@
 
 
 class BudgetLineItemStatus(Enum):
-    DRAFT = auto()
-    PLANNED = auto()
-    IN_EXECUTION = auto()
-    OBLIGATED = auto()
-
+    def __str__(self):
+            return str(self.value)
+    DRAFT = "Draft"
+    PLANNED = "Planned"
+    IN_EXECUTION = "In Execution"
+    OBLIGATED = "Obligated"
+
+def ConvertBLIStatusNameToPrettyString(BLIStatusName):
```

Comment:
This function should probably be a method on a class or in a utility function.  Also Python functions should not be in title case.

Reply:
This function is no longer title case and has been moved to a utility file.